index.js

```diff
@@ -0,0 +1,18 @@
+const bodyParser = require('body-parser')
+const jsonParser = bodyParser.json()
+const express = require('express')
+const app = express()
+
+const notes = []
+
+app.use(jsonParser)
+
+app.post('/notes/', (req, res) => {
+  notes.push(req.body)
+  notes.forEach((note, index) => {
+    note.id = index + 1
+  })
```

Comment:
I think you want to keep an `id` counter instead. For instance, if you have 5 notes, a user deletes a note, then adds a new note, that last one should have id of 6 and not 5.